docs/soyousegfaulted.md

```diff
@@ -0,0 +1,48 @@
+# So you got a segfault 
+
+While time and effort have been put into ensuring a bug-free experience, some edge cases may still arise that result in a segfault. 
+
+For such cases, we have prepared this very short documented to outline how you can best help us help you. 
+
+## Info to be provided
+
+Along with the info gathered by reading the section below, please also provide:
+
+1. lsif-clang version (commit/tag/etc)
+1. Output of `ldd <path to lsif-clang>`
+1. lsif-clang source (self-compiled/docker/prebuilt binary)
+
+## Accompanying Data
+
+### 1.  Is there a detailed stacktrace?
+
+The lsif-clang indexer includes the ability to emit a detailed stacktrace (when possible) as shown in the below screenshot if a segfault is encountered. Unfortunately it is not guaranteed that you will get this stacktrace, due to the complexity of the underlying clangd system. 
+
+![stacktrace in terminal](images/stacktrace.png)
+
+### 1. No stacktrace? Core dump time 
```

Comment:
What is the goal of the "1." prefix? I generally try to formulate headers as "How to ...". This makes it quite easy for users to skim over the document to find actionable advise.

```suggestion
### How to capture a core dump

It's helpful to include a core dump if the segfault produces no stack trace. The instructions to capture a core dump depend on the operating system.

....
```